compiler/ast/reports_internal.nim

```diff
@@ -44,7 +44,7 @@ type
     prefixdir*: string
     libpath*: string
     projectPath*: string
-    definedSymbols*: seq[string]
+    definedSymbols*: seq[tuple[key: string, val: string]]
```

Comment:
This was some quick and dirty debugging, although a cleaned up version of this would be good for introspection (separate PR).